deeprankcore/features/components.py

```diff
@@ -43,6 +52,6 @@ def add_features( # pylint: disable=unused-argument
                 node.features[Nfeat.VARIANTRES] = residue.amino_acid.onehot
                 node.features[Nfeat.DIFFCHARGE] = 0
                 node.features[Nfeat.DIFFSIZE] = 0
-                node.features[Nfeat.DIFFPOLARITY] = numpy.zeros(residue.amino_acid.polarity.onehot.shape)
+                node.features[Nfeat.DIFFPOLARITY] = np.zeros(residue.amino_acid.polarity.onehot.shape)

```

Comment:
Why does numpy have to be named np ?

Reply:
Very often it's imported as np, it's a simplification, and we changed it consistently (everywhere) in the codebase